src/main/java/de/hpi/bpt/argos/parsing/EventTypeParserImpl.java

```diff
@@ -0,0 +1,221 @@
+package de.hpi.bpt.argos.parsing;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import de.hpi.bpt.argos.core.Argos;
+import de.hpi.bpt.argos.storage.PersistenceAdapterImpl;
+import de.hpi.bpt.argos.storage.dataModel.attribute.type.TypeAttribute;
+import de.hpi.bpt.argos.storage.dataModel.attribute.type.TypeAttributeImpl;
+import de.hpi.bpt.argos.storage.dataModel.event.type.EventType;
+import de.hpi.bpt.argos.storage.dataModel.event.type.EventTypeImpl;
+import de.hpi.bpt.argos.storage.util.DataFile;
+import de.hpi.bpt.argos.storage.util.DataFileImpl;
+import de.hpi.bpt.argos.util.LoggerUtilImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * {@inheritDoc}
+ * This is the implementation.
+ */
+public class EventTypeParserImpl implements EventTypeParser {
+	private static final Logger logger = LoggerFactory.getLogger(EventTypeParserImpl.class);
+	private static final JsonParser jsonParser = new JsonParser();
+
+	private static final String JSON_NAME = "name";
+	private static final String JSON_TIMESTAMP = "timestamp";
+	private static final String JSON_ATTRIBUTES = "attributes";
+
+	private static EventTypeParser instance;
+
+	/**
+	 * This constructor hides the default public one to implement the singleton pattern.
+	 */
+	private EventTypeParserImpl() {
+
+		/**
+		 * File format from unicorn
+		 * Please note, that the given data-types are ignored.
+		 *
+		 * {
+		 *     "name":"EventTypeName",
+		 *     "timestamp":"EventTypeTimestamp",
+		 *     "attributes":{
+		 *			"eventTypeAttribute1":"STRING",
+		 *			"eventTypeAttribute2":"STRING",
+		 *			"eventTypeAttribute3":"INTEGER",
+		 *     }
+		 * }
+		 */
+	}
+
+	/**
+	 * This method returns the singleton instance of this class.
+	 * @return - the singleton instance of this class
+	 */
+	public static EventTypeParser getInstance() {
+		if (instance == null) {
+			instance = new EventTypeParserImpl();
+		}
+
+		return instance;
+	}
+
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public void loadEventTypes() {
+		String eventTypesDirectoryPath = Argos.getEventTypesDirectory();
+		if (eventTypesDirectoryPath.length() == 0) {
+			logger.info("event types directory not defined in properties");
```

Comment:
`LoggerUtil` is used for logging exceptions on trace-level. Since here is no exception, the `LoggerUtil` is not needed